gem/make/pe.mk

```diff
@@ -0,0 +1,23 @@
+SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
+include $(SELF_DIR)/common.mk
+
+NAMESPACE ?= artifactory.delivery.puppetlabs.net/pe-and-platform
+VERSION ?= `jq '."ubuntu-18.04-amd64"."pe-puppetserver"."version"' $(PWD)/tmp/enterprise-dist/packages.json 2> /dev/null | tr -d '"'`
+RELEASE ?= `jq '."ubuntu-18.04-amd64"."pe-puppetserver"."release"' $(PWD)/tmp/enterprise-dist/packages.json 2> /dev/null | tr -d '"'`
```

Comment:
I think `pe-puppetserver` will need to be replaced by some sort of var here?

Reply:
Oops yeah....missed that somehow